src/group_theory/group_action/pi.lean

```diff
@@ -33,9 +33,16 @@ instance has_smul {α : Type*} [Π i, has_smul α $ f i] :
 
 @[to_additive]
 lemma smul_def {α : Type*} [Π i, has_smul α $ f i] (s : α) : s • x = λ i, s • x i := rfl
-@[simp, to_additive]
+@[to_additive]
 lemma smul_apply {α : Type*} [Π i, has_smul α $ f i] (s : α) : (s • x) i = s • x i := rfl
 
+@[to_additive pi.has_smul]
+instance has_pow {β : Type*} [Π i, has_pow (f i) β] : has_pow (Π i, f i) β :=
+{ pow := λ x b i, (x i) ^ b }
```

Comment:
I think probably the thing to do is move this instance and the `has_smul` instance to `data.pi.algebra`, assuming it works there. If not, put them at the start of `group_action`.

Reply:
I have put them in `data.pi.algebra`